### ISSUE TITLE
Doc: add batch application

### ISSUE THREAD

**Issue by jbcaillau** [control-toolbox/OptimalControl.jl#246]
@agustinyabo can be for you 🙂

**Comment by jbcaillau** [comment]
@ocots can you pease add a pointer towards the (now removed) batch tutorial? was available, *e.g.*, during FGS 2024

**Comment by ocots** [comment]
[Here.](https://github.com/control-toolbox/OptimalControl.jl/blob/v0.9.3/docs/src/application-batch.md)

> [!TIP]
> Choose a tag instead of a branch in the code.
>
> <img width="919" alt="Capture d’écran 2024-07-23 à 11 44 38" src="https://github.com/user-attachments/assets/c1d144d9-978d-44d4-b39f-2d6b0a2699d2">

**Comment by ocots** [comment]
@jbcaillau We need a [template repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-template-repository) to facilitate adding applications.

**Comment by ocots** [comment]
Are the appli and the package templates different?

**Comment by jbcaillau** [comment]
right, let's do the same. and an app could become a package to be `used`.
